pom.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
There are some formatting changes which got added to this PR. Maybe if you build the project before pushing that will remove these formatting.

Reply:
Yeah it only formats the java files and not pom.xml